src/darwin/Framework/CHIPTests/TestHelpers/MTRTestResetCommissioneeHelper.h

```diff
@@ -14,10 +14,7 @@
  *    limitations under the License.
  */
 
-#import <Foundation/Foundation.h>
```

Comment:
Where is this getting dispatch_queue_t?

Reply:
Not from importing `<dispatch/dispatch.h>` apparently ;-)

Reply:
Like I said above, I did not realize that dispatch.h was a separate thing.  ;)

That said, sounds like the answer is "it's getting it via Matter.h, which includes MTRDefines.h, which includes Foundation.h".  And I guess having part of the public API contract for Matter.h be "includes Foundation.h" is reasonable, since it's defining ObjC types...